src/commonTest/kotlin/fr/acinq/lightning/channel/states/SpliceTestsCommon.kt

```diff
@@ -42,6 +42,12 @@ class SpliceTestsCommon : LightningTestSuite() {
         spliceIn(alice, bob, listOf(30_000.sat, 40_000.sat, 25_000.sat))
     }
 
+    @Test
+    fun `splice cpfp`() {
+        val (alice, bob) = reachNormal()
+        spliceCpfp(alice, bob)
```

Comment:
I added a couple splices in 020405d1, but the goal of this test is only to make sure that we supported a splice with no input and no output. It would be even more interesting to check feerates, but that requires testing at the peer level, which is quite hacky currently and needs a more general improvement.